test/intervals/TestIntervals.jl

```diff
@@ -183,7 +181,9 @@ facts("IntervalCollection") do
 
 
     context("Show") do
-        nullout = open("/dev/null", "w")
+        nullstream = @windows ? "NUL" : "/dev/null"
+        nullout = open(nullstream, "w")
```

Comment:
```
Bens-MacBook-Pro:~ axolotlfan9250$ julia
               _
   _       _ _(_)_     |  A fresh approach to technical computing
  (_)     | (_) (_)    |  Documentation: http://docs.julialang.org
   _ _   _| |_  __ _   |  Type "?help" for help.
  | | | | | | |/ _` |  |
  | | |_| | | | (_| |  |  Version 0.5.0-dev+1259 (2015-11-12 19:47 UTC)
 _/ |\__'_|_|_|\__'_|  |  Commit af36998 (45 days old master)
|__/                   |  x86_64-apple-darwin14.5.0

```
I generally pull master and run make on it when I install Julia, as a result I think I'm on v0.5.